LeetCodeAssistant/LeetCodeProblemRepository.swift

```diff
@@ -165,7 +196,7 @@ class LeetCodeProblemRepository {
                     assertionFailure()
                 }
                 
-                problem = APILeetCodeProblem(id: id, number: number, difficulty: difficulty, title: title)
+                problem = APILeetCodeProblem(id: id, number: number, difficulty: difficulty, title: title, description: "")
```

Comment:
This looks not good. If there's two static-states of "LeetCodeProblem" in the application, which means it should be represented as two structs/classes